Wrappers/Python/cil/utilities/display.py

```diff
@@ -103,277 +103,276 @@ def save(self,filename, **kwargs):
 
 class show1D(show_base):
     """
-    This creates and displays 1D plots of pixel values by slicing
-    multi-dimensional data.
+    Displays the 1D line profile of a N-dimensional dataset along a specified axis.
+
+    If multiple slices are provided, a plot will be generated for each slice.
 
-    The behaviour is as follows: if provided multiple datasets and a single
-    slice set (see first example below), one line plot will be generated
-    per dataset; if provided a single dataset and multiple sets of slices
-    (see second example below), one line plot will be generated per slice
-    set;  if provided multiple datasets and multiple slice sets, the
-    :math:`i`-th set of slices will apply to the :math:`i`-th dataset, with
-    a line plot generated in each case.
+    If multiple datasets are provided, the same slice will be applied to each dataset.
 
     Parameters
     ----------
-    data : DataContainer, list of DataContainer, tuple of DataContainer
-        Multi-dimensional data to be reduced to 1D.
-    slice_list : tuple, list of tuple or list of list of tuple, default=None
-        A tuple of (dimension, coordinate) pair, or a list, or nested list, of
-        such pairs for slicing `data` (default is None, which is only valid when 1D
-        data is passed)
-    label : 'default', str, list of str, None, default='default'
-        Label(s) to use in the plot's legend. Use `None` to suppress legend.
-    title : str, default None
-        A title for the plot
+    data : ndarray i.e. DataContainer, numpy.ndarray or a list of ndarrays
+        The data to be sliced and plotted. If a list of DataContainers is passed, they must all have the same shape.
+    slice_list : list of tuples, or list of list of tuple, optional.
+        Descibes the slice to be taken from the data to extract a 1D vector. If None, the centre line of each axis will be displayed.
+        Each slice should be a list of tuples, where each tuple is a (axis, index). Each axis can be described as a string or integer representing the axis to slice along.
+        To plot multiple slices, pass a list of slices.        
+    dataset_labels : list of str, optional
+        Overides the default legend labels for each dataset displayed in the plot legend. Must be a list of strings equal to the number of datasets.
```

Comment:
```suggestion
        Overrides the default legend labels for each dataset displayed in the plot legend. Must be a list of strings equal to the number of datasets.
```

Sorry, missed one! maybe do a find and replace on the file?